src/cli/mesos-ps

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Why change to python3?

Reply:
Sorry, that should not be part of this commit. I will remove it.

Reply:
To explain, for testing purpose, I compiled Mesos for AlmaLinux under arm64. As quick fix I changed the shebang because of error messages during the rpmbuild process. It should never be part of this commit. Sorry for that. 

But maybe it would be interesting, Mesos is working quite well under AlmaLinux (x86_64 and arm64). :-)